jobs/tsa/spec

```diff
@@ -78,3 +78,8 @@ properties:
     description: |
       Environment name to specify for errors emitted to Yeller.
     default: ""
+
+  tsa_atc_address:
```

Comment:
Let's make this `atc.url` instead. Properties should no longer be namespaced to their job with BOSH 2.0 now that they can be specified on individual jobs.